docs/src/styles/openpayments.css

```diff
@@ -13,6 +13,11 @@
   max-width: 48rem;
 }
 
+/* Odd one-off margin issue with expressive code (will hoist if seen again) */
```

Comment:
I'm not sure why there is a random insertion of script and link tag in between the container and the figure element containing the code but it is messing up the specificity of the margin-top for that block of code only. I'm unable to see it in other pages so I'm putting in this stop-gap in the OpenPayments-specific styles. If it ever shows up again in another docs site, I will elevate this fix to the shared theme library.
@melissahenderson if this is good enough for merging let me know, I also just pushed into your branch directly. please forgive the intrusion.